src/client/app/components/MapChartComponent.tsx

```diff
@@ -34,6 +32,8 @@ import { AreaUnitType, getAreaUnitConversion } from '../utils/getAreaUnitConvers
 import getGraphColor from '../utils/getGraphColor';
 import translate from '../utils/translate';
```

Comment:
I'm migrating translate to useTranslate as part of #1354 . Can you please migrate to useTranslate() from componentHooks.ts